src/Common/tests/System/Net/SslProtocolSupport.cs

```diff
@@ -11,7 +11,13 @@ namespace System.Net.Test.Common
 {
     public class SslProtocolSupport
     {
-        public const SslProtocols DefaultSslProtocols = SslProtocols.Tls12 | SslProtocols.Tls11 | SslProtocols.Tls;
+        private const SslProtocols Tls13Protocol = (SslProtocols)12288;
```

Comment:
same comment as above about magic number.

Reply:
I'm really not sure how much more of a named constant you'd want.  The number comes from the enum definition in 3.0.

Reply:
Perhaps just add a comment to the end of this line, i.e.

```c#
private const SslProtocols Tls13Protocol = (SslProtocols)12288; // PAL_SslProtocol_Tls13
```

It just makes it easier to figure it out rather than have to search all the other source files for the reference.